### DISCUSSION TITLE
How do I change git conifg?

### DISCUSSION THREAD

**Discussion by yingxiaoGuo** [community/community#26199] [Actions]
<pre><code>name: change lfs exclude

on: [push]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
      with:
        lfs: true
    - name: Run a one-line script
      run: git config --unset lfs.fetchexclude
</code></pre>
<p>I intend to change my git config every time it has a new push, and I try the above scripts. But every time it runs, it will return me:</p>
<pre><code>Run git config --unset lfs.fetchexclude
  git config --unset lfs.fetchexclude
  shell: /bin/bash -e {0}
##[error]Process completed with exit code 5.
</code></pre>
<p>Why? And what should I do? </p>
<p>Thanks in advance for your reply!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/yingxiaoguo">@yingxiaoguo</a> ,</p>
<p>How did you set ‘lfs.fetchexclude’ in your git config? Please cat the config file for confirmation( location, values…etc), please refer to my screenshot below. If i didn’t set ‘lfs.fetchexclude’ in config file, i can reproduce the same error with you.</p>
<p>Checked on my side, unset command works fine, code as below:</p>
<pre><code>name: change lfs exclude
on: [push]

jobs:
  build:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
      with:
        lfs: true
    - name: cat git config
      run: |
        cd .git
        cat config
    - name: Set the lfs.fetchexclude
      run: |
        git config lfs.fetchexclude "media/excessive"
    - name: cat git config
      run: |
        cd .git
        cat config
    - name: check git config after checkout
      run: |
        git config -l
    - name: unset
      run: |
        git config --unset lfs.fetchexclude
    - name: check git config after unset
      run: |
        git config -l
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4fd11bf3ef348f666dad139ea5eff41d5ce752d9.png" data-download-href="/uploads/short-url/bo5JxzCQlkimApbL5WXz4DAlzJL.png?dl=1" title="unset.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181096037-5bd28771-688f-4b39-8c10-54f027a99f60.png" alt="unset.png" data-base62-sha1="bo5JxzCQlkimApbL5WXz4DAlzJL" width="609" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4fd11bf3ef348f666dad139ea5eff41d5ce752d9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">unset.png</span><span class="informations">625×513 24.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks.</p>

**Comment by yingxiaoGuo** [comment]
<p>Hi, Weide</p>
<p>I think I get what’s going on here. I was intended to use this script to keep my remote config different from my local config, but I didn’t realize that local config won’t be pushed to the remote so I encountered this error. </p>
<p>Thanks for your reply!</p>

**Comment by weide-zhou** [comment]
<p>Hi <a href="/t5/user/viewprofilepage/user-id/70154">@yingxiaoGuo</a> ,</p>
<p>Thanks for your mark!</p>
